Testscripts/Windows/LIS-Check-HypervDaemons-Files-Status.ps1

```diff
@@ -0,0 +1,130 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+<#
+.Synopsis
+    This script tests hyper-v daemons service status and files.
+
+.Description
+    The script will enable "Guest services" in "Integration Service" if it
+    is disabled, then execute "Hyperv_Daemons_Files_Status" to check hypervkvpd,
+    hypervvssd,hypervfcopyd status and default files.
+'
+#>
+
+param([String] $TestParams)
+
+function Main {
+    param (
+        $VMName,
+        $HvServer,
+        $Ipv4,
+        $VMPort,
+        $VMUserName,
+        $VMPassword,
+        $RootDir,
+        $TestParams
+    )
+
+    $gsi = $null
+    $remoteScript = "LIS-Check-HypervDaemons-Files-Status.sh"
+    #######################################################################
+    #
+    #	Main body script
+    #
+    #######################################################################
+
+    # Checking the input arguments
+    if (-not $VMName) {
+        LogErr  "VM name is null!"
+        return "FAIL"
+    }
+
+    if (-not $HvServer) {
+        LogErr "hvServer is null!"
+        return "FAIL"
+    }
+    #
+    # Checking the mandatory testParams. New parameters must be validated here.
+    #
+    $params = $TestParams.Split(";")
+    foreach ($p in $params) {
+        $fields = $p.Split("=")
+
+        if ($fields[0].Trim() -eq "RootDir") {
```

Comment:
You can delete this line. RootDir is already sent as a param to Main and it's not in TestParams. This is a "lisa" specific code that it's not used anymore in lisav2